README.md

```diff
@@ -125,15 +125,15 @@ pre-commit:
 ## Custom file list
 
 Lefthook can be even more specific in selecting files.
-If you want to choose diff of all changed files between the current branch and master branch you can do it this way:
+If you want to choose all files changed locally but pushed to GitHub yet.
 
 ```yml
 # lefthook.yml
 
 pre-push:
   commands:
     frontend-style:
-      files: git diff --name-only master # custom list of files
+      files: git diff --name-only origin # custom list of files
```

Comment:
```suggestion
      files: git diff --name-only @{push} || git diff --name-only master # custom list of files
```

Reply:
This command has fallback to master. Tested in local console :)

Reply:
Do we need a fallback to master? My suggestion:

1. If we are in `master` we need to compare with `origin/master`
2. If we are in `feature` we need to compare with `origin/feature`

Reply:
I'm not sure. Sometimes there is no upstream for the current branch (for example `git checkout -b`). In such a case the check will always fail :(

cc @Arkweid

Reply:
I'm not sure that we need something complicated in this example.
I think it is enough to make it clear that you can throw any command on this line, so long as it returns the list of files.

Reply:
Yeap, maybe you are right and we have a bigger problem here.

What if we change the docs:
1. We will start with a real useful copy-paste example.
2. In the next chapters, we will explain how to create complicated things.

This issue is about my problem: I have `logux/logux` project where I want to use Lefthook and I do not know what to write in config. The most popular thing: check all changed files before `git push`. The current docs don’t tell me how to do it. We need to improve them to make migration easier.